### ISSUE TITLE
[protobuf:x86-windows] build failure

### ISSUE THREAD

**Issue by aleneum** [microsoft/vcpkg#13426]
**Host Environment**
- OS: Windows 10 Version 2004
- Compiler: v142 toolset

**To Reproduce**
`./vcpkg install protobuf:x86-windows`

**Failure logs** 
```
[DEBUG] Feature flag 'binarycaching' unset
[DEBUG] Feature flag 'manifests' unset
[DEBUG] Feature flag 'compilertracking' unset
[DEBUG] Feature flag 'registries' unset
[DEBUG] Using vcpkg-root: C:\Users\alneuman\workspace\vcpkg
[DEBUG] Using installed-root: C:\Users\alneuman\workspace\vcpkg\installed
[DEBUG] Using buildtrees-root: C:\Users\alneuman\workspace\vcpkg\buildtrees
[DEBUG] Using downloads-root: C:\Users\alneuman\workspace\vcpkg\downloads
[DEBUG] Using packages-root: C:\Users\alneuman\workspace\vcpkg\packages
[DEBUG] Using scripts-root: C:\Users\alneuman\workspace\vcpkg\scripts
Your feedback is important to improve Vcpkg! Please take 3 minutes to complete our survey by running: vcpkg contact --survey
[DEBUG] Default binary cache path is: C:\Users\alneuman\AppData\Local\vcpkg/archives
Computing installation plan...
[DEBUG] Found path: C:\Program Files\CMake\bin/cmake.exe
[DEBUG] CreateProcessW("C:\Program Files\CMake\bin/cmake.exe" --version)
[DEBUG] cmd_execute_and_stream_data() returned 0 after    27589 us
[DEBUG] CreateProcessW("C:\Program Files\CMake\bin/cmake.exe" -DVCPKG_ROOT_DIR=C:/Users/alneuman/workspace/vcpkg -DPACKAGES_DIR=C:/Users/alneuman/workspace/vcpkg/packages -DBUILDTREES_DIR=C:/Users/alneuman/workspace/vcpkg/buildtrees -D_VCPKG_INSTALLED_DIR=C:/Users/alneuman/workspace/vcpkg/installed -DDOWNLOADS=C:/Users/alneuman/workspace/vcpkg/downloads -DVCPKG_MANIFEST_INSTALL=OFF -P "C:\Users\alneuman\workspace\vcpkg\buildtrees\0.vcpkg_tags.cmake")
[DEBUG] cmd_execute_and_stream_data() returned 0 after    19856 us
The following packages will be built and installed:
    protobuf[core]:x86-windows
[DEBUG] CreateProcessW("C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe" -all -prerelease -legacy -products * -format xml)
[DEBUG] cmd_execute_and_stream_data() returned 0 after    23672 us
Detecting compiler hash for triplet x86-windows...
[DEBUG] Found path: C:\Program Files\Git\cmd/git.exe
[DEBUG] CreateProcessW("C:\Program Files\Git\cmd/git.exe" --version)
[DEBUG] cmd_execute_and_stream_data() returned 0 after    34033 us
[DEBUG] CreateProcessW(cmd /c ""C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Auxiliary\Build\vcvarsall.bat" -vcvars_ver=14.2 amd64_x86   2>&1 <NUL" & echo cdARN4xjKueKScMy9C6H& set)
[DEBUG] cmd_execute_and_stream_data() returned 0 after  1581974 us
[DEBUG] CreateProcessW("C:\Program Files\CMake\bin/cmake.exe" -DCURRENT_PORT_DIR=C:/Users/alneuman/workspace/vcpkg/scripts/detect_compiler -DCURRENT_BUILDTREES_DIR=C:/Users/alneuman/workspace/vcpkg/buildtrees/detect_compiler -DCURRENT_PACKAGES_DIR=C:/Users/alneuman/workspace/vcpkg/packages/detect_compiler_x86-windows -DCMD=BUILD -DTARGET_TRIPLET=x86-windows "-DTARGET_TRIPLET_FILE=C:\Users\alneuman\workspace\vcpkg\triplets\x86-windows.cmake" -DVCPKG_PLATFORM_TOOLSET=v142 -DDOWNLOADS=C:/Users/alneuman/workspace/vcpkg/downloads -DVCPKG_CONCURRENCY=17 "-DGIT=C:/Program Files/Git/cmd/git.exe" -DVCPKG_ROOT_DIR=C:/Users/alneuman/workspace/vcpkg -DPACKAGES_DIR=C:/Users/alneuman/workspace/vcpkg/packages -DBUILDTREES_DIR=C:/Users/alneuman/workspace/vcpkg/buildtrees -D_VCPKG_INSTALLED_DIR=C:/Users/alneuman/workspace/vcpkg/installed -DDOWNLOADS=C:/Users/alneuman/workspace/vcpkg/downloads -DVCPKG_MANIFEST_INSTALL=OFF -P "C:\Users\alneuman\workspace\vcpkg\scripts\ports.cmake")
[DEBUG] -- Configuring x86-windows
[DEBUG] [1/1] cmd /c "cd .. && "C:/Program Files/CMake/bin/cmake.exe" "C:/Users/alneuman/workspace/vcpkg/scripts/detect_compiler" "-DCMAKE_MAKE_PROGRAM=C:/Users/alneuman/workspace/vcpkg/downloads/tools/ninja/1.10.0-windows/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/Users/alneuman/workspace/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x86-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/Users/alneuman/workspace/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x86" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/Users/alneuman/workspace/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/Users/alneuman/workspace/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=C:/Users/alneuman/workspace/vcpkg/packages/detect_compiler_x86-windows""
[DEBUG] #COMPILER_HASH#b78026da3af76d466d112342f120edab261c285b
[DEBUG] -- Configuring done
[DEBUG] -- Generating done
[DEBUG] CMake Warning:
[DEBUG]   Manually-specified variables were not used by the project:
[DEBUG]
[DEBUG]     BUILD_SHARED_LIBS
[DEBUG]     CMAKE_INSTALL_BINDIR
[DEBUG]     CMAKE_INSTALL_LIBDIR
[DEBUG]     VCPKG_TARGET_ARCHITECTURE
[DEBUG]
[DEBUG]
[DEBUG] -- Build files have been written to: C:/Users/alneuman/workspace/vcpkg/buildtrees/detect_compiler/x86-windows-rel
[DEBUG]
[DEBUG] cmd_execute_and_stream_data() returned 0 after  1398700 us
[DEBUG]
[DEBUG] Detecting compiler hash for triplet x86-windows: b78026da3af76d466d112342f120edab261c285b
[DEBUG] <abientries for protobuf:x86-windows>
[DEBUG]   CONTROL|8caf72a6847b48547e46054499e54a4aa036a784
[DEBUG]   cmake|3.18.2
[DEBUG]   execute_process|7b25b946cf0f36a6995cfe043fc9a1e225a3548c
[DEBUG]   features|core
[DEBUG]   fix-android-log.patch|b3dfd8e9bf9678488312b7df306f747a62dba128
[DEBUG]   fix-static-build.patch|e38420fa529a16112b01f6d54f460c5eaec1702d
[DEBUG]   fix-uwp.patch|8afada3f93623321aa2b9fb1e369b14361f003d5
[DEBUG]   portfile.cmake|ed0e5e5d83801635a8fa5e4b9e5090896d6d88b6
[DEBUG]   post_build_checks|2
[DEBUG]   powershell|2e410fc5e429fc6ad478976eff3efcffe9d53a0620c7c31c2b8bfe1f0976cade74632fc202bf58959cec111f03b824ba42ad4d74c8a66d67ec21bbfe9b9c626d
[DEBUG]   triplet|e8dc96fd479b53bb0e6748625827c8268017e35a-a1c0eabb0c5177b6a8fb97a58ae398880c47b352-b78026da3af76d466d112342f120edab261c285b
[DEBUG]   vcpkg_check_features|bf530963ff43fceb8cac60d441e635c4233ba35c
[DEBUG]   vcpkg_configure_cmake|d9e525769b8519c26e4c0446242434961527d33b
[DEBUG]   vcpkg_copy_pdbs|4bbc15802daf8f9f8a92794c10b0a89cfca1e742
[DEBUG]   vcpkg_copy_tools|f6400daff6e1d17e058a8b8c24b5178f66f82c10
[DEBUG]   vcpkg_find_acquire_program|4234efa37fa8b36cbc6476052726ae1480c81080
[DEBUG]   vcpkg_fixup_pkgconfig|9cb19196ba10f50955350e422a9004ea2d46e0dc
[DEBUG]   vcpkg_from_git|13f63ff436e043b1297055ca7d59776d9e925cd1
[DEBUG]   vcpkg_from_github|4624dffdf2b8d85ffdfde0850369fa96abdf0bb8
[DEBUG]   vcpkg_install_cmake|75d1079d6f563d87d08c8c5f63b359377aa19a6e
[DEBUG]   vcpkg_replace_string|99deefc316db487a541ba0b25d6249a0b86e2186
[DEBUG] </abientries>
Starting package 1/1: protobuf:x86-windows
Building package protobuf[core]:x86-windows...
Could not locate cached archive: C:\Users\alneuman\AppData\Local\vcpkg/archives\7f\7f4c033143a4234386dc2dd2bbc08b20cdaf03f1.zip
[DEBUG] CreateProcessW("C:\Program Files\CMake\bin/cmake.exe" -DPORT=protobuf -DCURRENT_PORT_DIR=C:/Users/alneuman/workspace/vcpkg/ports/protobuf -DVCPKG_USE_HEAD_VERSION=0 -D_VCPKG_NO_DOWNLOADS=0 -D_VCPKG_DOWNLOAD_TOOL=BUILT_IN -D_VCPKG_EDITABLE=0 -DFEATURES=core "-DALL_FEATURES=zlib;" -DCMD=BUILD -DTARGET_TRIPLET=x86-windows "-DTARGET_TRIPLET_FILE=C:\Users\alneuman\workspace\vcpkg\triplets\x86-windows.cmake" -DVCPKG_PLATFORM_TOOLSET=v142 -DDOWNLOADS=C:/Users/alneuman/workspace/vcpkg/downloads -DVCPKG_CONCURRENCY=17 "-DGIT=C:/Program Files/Git/cmd/git.exe" -DVCPKG_ROOT_DIR=C:/Users/alneuman/workspace/vcpkg -DPACKAGES_DIR=C:/Users/alneuman/workspace/vcpkg/packages -DBUILDTREES_DIR=C:/Users/alneuman/workspace/vcpkg/buildtrees -D_VCPKG_INSTALLED_DIR=C:/Users/alneuman/workspace/vcpkg/installed -DDOWNLOADS=C:/Users/alneuman/workspace/vcpkg/downloads -DVCPKG_MANIFEST_INSTALL=OFF -P "C:\Users\alneuman\workspace\vcpkg\scripts\ports.cmake")
-- Using cached C:/Users/alneuman/workspace/vcpkg/downloads/protocolbuffers-protobuf-fde7cf7358ec7cd69e8db9be4f1fa6a5c431386a.tar.gz
-- Cleaning sources at C:/Users/alneuman/workspace/vcpkg/buildtrees/protobuf/src/a5c431386a-c9deea9e31.clean. Use --editable to skip cleaning for the packages you specify.
-- Using source at C:/Users/alneuman/workspace/vcpkg/buildtrees/protobuf/src/a5c431386a-c9deea9e31.clean
-- Configuring x86-windows
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:83 (message):
    Command failed: ninja -v
    Working Directory: C:/Users/alneuman/workspace/vcpkg/buildtrees/protobuf/x86-windows-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      C:\Users\alneuman\workspace\vcpkg\buildtrees\protobuf\config-x86-windows-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_configure_cmake.cmake:305 (vcpkg_execute_required_process)
  ports/protobuf/portfile.cmake:47 (vcpkg_configure_cmake)
  scripts/ports.cmake:79 (include)


[DEBUG] cmd_execute_and_stream_data() returned 1 after  2744678 us
Error: Building package protobuf:x86-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: protobuf:x86-windows
  Vcpkg version: 2020.06.15-nohash

Additionally, attach any relevant sections from the log files above.
[DEBUG] C:\Users\alneuman\workspace\vcpkg\toolsrc\src\vcpkg\install.cpp(493)
[DEBUG] Exiting after 5878832 us (5869978 us)
```

### C:\Users\alneuman\workspace\vcpkg\buildtrees\protobuf\config-x86-windows-out.log

```
[1/2] cmd /c "cd ../../x86-windows-dbg && "C:/Program Files/CMake/bin/cmake.exe" "C:/Users/alneuman/workspace/vcpkg/buildtrees/protobuf/src/a5c431386a-c9deea9e31.clean/cmake" "-Dprotobuf_BUILD_SHARED_LIBS=ON" "-Dprotobuf_MSVC_STATIC_RUNTIME=OFF" "-Dprotobuf_BUILD_TESTS=OFF" "-DCMAKE_INSTALL_CMAKEDIR:STRING=share/protobuf" "-Dprotobuf_BUILD_PROTOC_BINARIES=ON" "-Dprotobuf_WITH_ZLIB=OFF" "-DCMAKE_MAKE_PROGRAM=C:/Users/alneuman/workspace/vcpkg/downloads/tools/ninja/1.10.0-windows/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/Users/alneuman/workspace/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x86-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/Users/alneuman/workspace/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x86" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/Users/alneuman/workspace/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/Users/alneuman/workspace/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=C:/Users/alneuman/workspace/vcpkg/packages/protobuf_x86-windows/debug""
FAILED: ../../x86-windows-dbg/CMakeCache.txt 
cmd /c "cd ../../x86-windows-dbg && "C:/Program Files/CMake/bin/cmake.exe" "C:/Users/alneuman/workspace/vcpkg/buildtrees/protobuf/src/a5c431386a-c9deea9e31.clean/cmake" "-Dprotobuf_BUILD_SHARED_LIBS=ON" "-Dprotobuf_MSVC_STATIC_RUNTIME=OFF" "-Dprotobuf_BUILD_TESTS=OFF" "-DCMAKE_INSTALL_CMAKEDIR:STRING=share/protobuf" "-Dprotobuf_BUILD_PROTOC_BINARIES=ON" "-Dprotobuf_WITH_ZLIB=OFF" "-DCMAKE_MAKE_PROGRAM=C:/Users/alneuman/workspace/vcpkg/downloads/tools/ninja/1.10.0-windows/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/Users/alneuman/workspace/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x86-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/Users/alneuman/workspace/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x86" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/Users/alneuman/workspace/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/Users/alneuman/workspace/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=C:/Users/alneuman/workspace/vcpkg/packages/protobuf_x86-windows/debug""
CMake Error: The source directory "C:/Users/alneuman/workspace/vcpkg/buildtrees/protobuf/src/a5c431386a-c9deea9e31.clean/cmake" does not exist.
Specify --help for usage, or press the help button on the CMake GUI.
[2/2] cmd /c "cd .. && "C:/Program Files/CMake/bin/cmake.exe" "C:/Users/alneuman/workspace/vcpkg/buildtrees/protobuf/src/a5c431386a-c9deea9e31.clean/cmake" "-Dprotobuf_BUILD_SHARED_LIBS=ON" "-Dprotobuf_MSVC_STATIC_RUNTIME=OFF" "-Dprotobuf_BUILD_TESTS=OFF" "-DCMAKE_INSTALL_CMAKEDIR:STRING=share/protobuf" "-Dprotobuf_BUILD_PROTOC_BINARIES=ON" "-Dprotobuf_WITH_ZLIB=OFF" "-DCMAKE_MAKE_PROGRAM=C:/Users/alneuman/workspace/vcpkg/downloads/tools/ninja/1.10.0-windows/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/Users/alneuman/workspace/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x86-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/Users/alneuman/workspace/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x86" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/Users/alneuman/workspace/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/Users/alneuman/workspace/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=C:/Users/alneuman/workspace/vcpkg/packages/protobuf_x86-windows""
FAILED: ../CMakeCache.txt 
cmd /c "cd .. && "C:/Program Files/CMake/bin/cmake.exe" "C:/Users/alneuman/workspace/vcpkg/buildtrees/protobuf/src/a5c431386a-c9deea9e31.clean/cmake" "-Dprotobuf_BUILD_SHARED_LIBS=ON" "-Dprotobuf_MSVC_STATIC_RUNTIME=OFF" "-Dprotobuf_BUILD_TESTS=OFF" "-DCMAKE_INSTALL_CMAKEDIR:STRING=share/protobuf" "-Dprotobuf_BUILD_PROTOC_BINARIES=ON" "-Dprotobuf_WITH_ZLIB=OFF" "-DCMAKE_MAKE_PROGRAM=C:/Users/alneuman/workspace/vcpkg/downloads/tools/ninja/1.10.0-windows/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/Users/alneuman/workspace/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x86-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/Users/alneuman/workspace/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x86" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/Users/alneuman/workspace/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/Users/alneuman/workspace/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=C:/Users/alneuman/workspace/vcpkg/packages/protobuf_x86-windows""
CMake Error: The source directory "C:/Users/alneuman/workspace/vcpkg/buildtrees/protobuf/src/a5c431386a-c9deea9e31.clean/cmake" does not exist.
Specify --help for usage, or press the help button on the CMake GUI.
ninja: build stopped: subcommand failed.
```

**Additional context**
Add any other context about the problem here, such as what you have already tried to resolve the issue.

**Comment by sfhacker** [comment]
Are we **STILL** having issues building protobuf?

![image](https://user-images.githubusercontent.com/25817529/92589679-41462900-f2de-11ea-8d6f-c6ef9c21f724.png)

**Comment by aleneum** [comment]
Looks like it. I checked out the tagged version `2020.07` and could build protobuf without any issues.

**Comment by JackBoosY** [comment]
`CMake Error: The source directory "C:/Users/alneuman/workspace/vcpkg/buildtrees/protobuf/src/a5c431386a-c9deea9e31.clean/cmake" does not exist.`

Can you confirm that the path _C:/Users/alneuman/workspace/vcpkg/buildtrees/protobuf/src/a5c431386a-c9deea9e31.clean/cmake_ exists?

**Comment by aleneum** [comment]
Hello @JackBoosY ,

> Can you confirm that the path C:/Users/alneuman/workspace/vcpkg/buildtrees/protobuf/src/a5c431386a-c9deea9e31.clean/cmake exists?

the folder was completely empty. I built `.\vcpkg.exe install grpc --triplet arm64-windows-static` first and was asked to install `protobuf:x86-windows` which failed. I cloned vcpkg multiple times to make sure that there are not side effects.

But I cannot reproduce it today even though I cannot see changes that made this behavior go away. So it was probably my machine or me :/. Sorry, for the inconvenience. Awesome project btw. 👍 !
